STL_Extension/include/CGAL/Concurrent_compact_container.h

```diff
@@ -233,6 +233,11 @@ class Concurrent_compact_container
     clear();
   }
 
+  bool is_used(const_iterator ptr) const
+  {
+    return (type(&*ptr)==USED);
+  }
+
```

Comment:
It's fine since the documentation says: "An object of the class `Concurrent_compact_container` 
is a container [...] which allows to call `insert` and `erase` operations concurrently.
Other operations are not concurrency-safe."
And `is_used` is meant to be called when parsing the content of the container, which is a read-only operation which is already not possible when other threads are modifying the container (it wouldn't make much sense).